help/index.php

```diff
@@ -0,0 +1,164 @@
+	<?=common_top()?>
+
+	<meta charset="utf-8">
+	<title>Swamp Servers - Commands</title>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<style type="text/css">
```

Comment:
I already have a stylesheet. I don't want any of this unless there's a good reason for it.